### PR TITLE
Sync the site footer with the main website

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This essentially applies https://github.com/srobo/website/pull/360 and any other recent changes.

Fixes https://github.com/srobo/docs/issues/327

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -48,8 +49,11 @@ <h4>&nbsp;</h4>
     <div class="column d-6-12 text-right">
       <ul class="social-media-links">
         {% for account in site.social %}
-        <li><a href="{{ account.link }}"><i class="fa fa-{{ account.icon }}" aria-hidden="true"
-              title="{{ account.title }}"></i></a></li>
+        <li>
+          <a aria-label="{{ account.title }}" href="{{ account.link }}">
+            <i class="fa-brands fa-{{ account.icon }}" aria-hidden="true" title="{{ account.title }}"></i>
```
Comment (sedders123) [inline _includes/footer.html:54]: The `fa-brands` change here was to allow us to upgrade to FontAwesome 6 so we could add TikTok to the social links. Do we change back to `fa` here or also upgrade to FontAwesome 6 on the docs site + add TikTok to socials as part of this PR?

Comment (sedders123) [reply _includes/footer.html:54]: For reference https://github.com/srobo/website/pull/434/files

Comment (PeterJCLaw) [reply _includes/footer.html:54]: Ah, right. I'll change this back to `fa` and we can handle that separately unless you think it would be easy to roll in?

Comment (sedders123) [reply _includes/footer.html:54]: Think the upgrade might be worth its own PR. Changes may required on other pages/parts of the site

Comment (PeterJCLaw) [reply _includes/footer.html:54]: #357 handles the font-awesome update.

**Review by sedders123 — APPROVED** [review]

**Pull request merged**